doc/versioning.md

```diff
@@ -0,0 +1,51 @@
+# Versioning strategy
```

Comment:
Maybe make the title "Override the Default In-App Payments SDK Dependency Version"